website/templates/emails/invite_preprints_branded.txt.mako

```diff
@@ -4,11 +4,15 @@
 
 Hello ${fullname},
 
-You have been added by ${referrer.fullname} as a contributor to the preprint "${node.title}" on ${branded_service.name}, powered by the Open Science Framework. To set a password for your account, visit:
+You have been added by ${referrer.fullname} as a contributor to the ${branded_service.preprint_word} "${node.title}" on ${branded_service.name}, powered by the Open Science Framework. To set a password for your account, visit:
 
 ${claim_url}
 
-Once you have set a password, you will be able to make contributions to "${node.title}" and create your own preprints. You will automatically be subscribed to notification emails for this preprint. Each preprint is associated with a project on the Open Science Framework for managing the preprint.  To change your email notification preferences, visit your project or your user settings: ${settings.DOMAIN + "settings/notifications/"}
+%if branded_service.name == 'Thesis Commons':
+Once you have set a password, you will be able to make contributions to "${node.title}" and create your own theses. You will automatically be subscribed to notification emails for this ${branded_service.preprint_word}. Each ${branded_service.preprint_word} is associated with a project on the Open Science Framework for managing the ${branded_service.preprint_word}.  To change your email notification preferences, visit your project or your user settings: ${settings.DOMAIN + "settings/notifications/"}
```

Comment:
No, this won't work. We may one day have another word that won't be modifiable into a plural just by adding `s` to the end. Let's do something like modifying https://github.com/CenterForOpenScience/osf.io/blob/aef8d08d588d54148f4cfa013fc496bbeae93996/osf/models/preprint_provider.py#L43-L48 to have an extra item in the tuple for the plural version. Then you can derive the choice field from that and make a property (which you can use in the emails) that will give you the plural version for any provider. Or if you think of a better way to do the same sort of thing without having two lists that have to be maintained, go for that.